### ISSUE TITLE
Threshold for common path prefix

### ISSUE THREAD

**Issue by smichaelsen** [martin-helmich/typo3-typoscript-lint#36]
Right now a warning is raised if you have two assignments which share a common path prefix:

````
lib.foo = bar
lib.bax = baz
````
I'd like to see a configurable threshold here. Normally we accept having two assignments with a common path prefix but begin nesting as soon as we have 3 assignments.
